ceph.spec.in

```diff
@@ -1465,6 +1465,10 @@ install -m 644 -D monitoring/ceph-mixin/prometheus_alerts.yml %{buildroot}/etc/p
 %py_byte_compile %{__python3} %{buildroot}%{python3_sitelib}
 %endif
 
+# built binaries are no longer necessary at this point,
+# but are consuming ~17GB of disk in the build environment
+rm -rf build
```

Comment:
might want to use `%{_vpath_builddir}` instead. see also 6de97d44342ac2a2856f427c3ac648bb6f50abcc and  https://docs.fedoraproject.org/en-US/packaging-guidelines/vpath/

Reply:
Thanks for the pointer, will adjust.